src/backend/CommandBufferStateTracker.cpp

```diff
@@ -352,6 +354,18 @@ namespace backend {
         return true;
     }
 
+    bool CommandBufferStateTracker::EnsureTextureUsage(TextureBase* texture, nxt::TextureUsageBit usage) {
```

Comment:
nit: EnsureTextureUsage is weird. Maybe "PerformImplicitTextureUsageTransition"?